packaging/Dockerfile.mockbuild

```diff
@@ -1,47 +1,74 @@
 #
-# Copyright 2018-2022 Intel Corporation
+# Copyright 2018-2024 Intel Corporation
 #
 # 'recipe' for Docker to build an RPM
 #
 
 # Pull base image
-FROM fedora:latest
+ARG FVERSION=latest
+FROM fedora:$FVERSION
+# Needed for later use of FVERSION
+ARG FVERSION
```

Comment:
What is the purpose of ```ARG FVERSION``` when there is ```ARG FVERSION=latest``` above?

Reply:
It is a documented feature of Dockerfiles.   The first ARG statements with default values above the FROM statement are only visible to the FROM statement.   The ARG statement with the same name, makes that ARG value visible to the rest of the Dockerfile.